src/Behavioral.Automation/Bindings/DropdownBinding.cs

```diff
@@ -77,6 +87,39 @@ public void CheckMultipleSelectedValues([NotNull] IMultiSelectDropdownWrapper wr
             wrapper.SelectedValuesTexts.Should().BeEquivalentTo(values.Rows.Select(x => x.Values.Single()));
         }
 
+        [Given("the \"(.*?)\" value (is|become) (enabled|disabled) in (.*?)")]
+        [Then("the \"(.*?)\" value should (be|become) (enabled|disabled) in (.*?)")]
+        public void CheckValueInDropdownIsEnabled([NotNull] string value, [NotNull] AssertionBehavior behavior,
+            bool enabled, IDropdownWrapper wrapper)
+        {
+            _attributeBinding.CheckElementIsDisabled(wrapper.Elements.Single(x => x.Text == value), behavior, enabled);
+        }
+
+        [Given("the following values (are|become) (enabled|disabled) in (.*?):")]
```

Comment:
Please, ensure that this binding doesn't overlap with inside binding.

Reply:
It shouldn't because we're using "the" and it doesn't contain word "inside"